.gitignore

```diff
@@ -87,8 +87,13 @@ rebel.xml
 # Ignoring package-lock.json
 package-lock.json
 
+
+# Ignoring yarn.lock
+**/yarn.lock
+
 # Ignoring protobuf related files
 scalapb/scalapb
+**/scalapb/scalapb
```

Comment:
it's better if we revert those two changes. `yarn.lock` is needed to be version controlled. also `scalapb/scalapb` and `**/scalapb/scalapb` are equivelent

Reply:
Thanks. I'll fix that.

Reply:
Fixed.